lms/views/decorators/h_api.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+"""View decorators for working with the h API."""
+
+from __future__ import unicode_literals
+
+import json
+import requests
+
+from pyramid.httpexceptions import HTTPBadGateway
+from pyramid.httpexceptions import HTTPBadRequest
+from pyramid.httpexceptions import HTTPGatewayTimeout
+
+from lms import util
+from lms.util import MissingToolConsumerIntanceGUIDError
+from lms.util import MissingUserIDError
+
+
+def create_h_user(wrapped):  # noqa: MC0001
```

Comment:
Would you mind expanding the docstring to document the `wrapped` param and the possible return values of this function?